deel-2-flexbox-layout/opdracht-3/styles/styles.css

```diff
@@ -1 +1,74 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+*, *::before, *::after {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+    border: 3px solid lightgray;
+    /*flex-basis: auto;*/
+}
+/*//hier zeg ik alleen wat wil de nav doen? het is al een kind van de body die */
+/*in row staat dus daar hoef ik niks aan te doen */
+.col-1{
+    /*display: flex;*/
+    /*flex-direction: column;*/
+    flex-basis: 20vw;
+    flex-grow:0;
+    flex-shrink:0;
+    border: 3px dashed green;
+    /*height: 400px;*/
+}
+
+/*hier wil je van richting veranderen dus hier ga je weer die display flex inzetten */
+/*in de html staat een div om de elementen heen t/m footer die maak je dus de parent*/
+.col-2{
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    flex-shrink:1;
+    border: 3px dashed black;
+    /*flex-basis: 1400px;*/
+    /*height: 400px;*/
+    /*flex-direction: row;*/
+}
+
+/*de header is een column dus de basis zegt nu hoever naar beneden*/
+header {
+    flex-basis: 10rem;
```

Comment:
Ik weet niet of het gebruik van _ rem_  (en _em_) handig is om maten van niet-font dingen vast te leggen. Je zegt hier: maak de _header_ 10x zo groot als de grootte van mijn _root font size_ (default 16px, dus 10 * 16px = 160px.). Maar als je dan later besluit om je font size te veranderen, dan veranderen ook opeens andere elementen op je pagina zoals header en footer?

Kan misschien wel handig voor elementen die tekst bevatten trouwens! :D